### ISSUE TITLE
Comment box is flashing while reloading the post page

### ISSUE THREAD

**Issue by dabbit** [auralite/web#19]
Comment box is flashing while reloading the post page

![Screenshot](https://user-images.githubusercontent.com/68646469/89119158-9d40b500-d4c9-11ea-9086-4d77676b4183.png)

After that, it displays the blank card

![Screenshot](https://user-images.githubusercontent.com/68646469/89119176-b5183900-d4c9-11ea-8ef9-94f61185d6e7.png)

**Comment by m1guelpf** [comment]
Yeah, this is because the layout is rendered on the server by default (which assumes an authenticated state), and it can take a second to update the state. A lot of apps actually do this, and I don't see it as a big issue (although I should definitely look into getting rid of the white box too).
